routing/cross_routing_context.hpp

```diff
@@ -70,6 +70,8 @@ struct OutgoingCrossNode
   void Save(Writer & w) const;
 
   size_t Load(Reader const & r, size_t pos, size_t adjacencyIndex);
+
+  m2::RectD const GetLimitRect() const { return m2::RectD(m_point.lat, m_point.lon, m_point.lat, m_point.lon); }
```

Comment:
Err, m2::RectD should be in mercator coordinates, not in lat-lons. PointD and RectD should always be in mercator, and never in lat-lons.